### ISSUE TITLE
SVG are not included in printings (paper and PDF)

### ISSUE THREAD

**Issue by Norbert80** [ONLYOFFICE/DesktopEditors#1624]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package)

### Version information

Version 8.1.0.169 (deb)

### Expected Behavior

Printings (paper and PDF) should contain SVG.

### Actual Behavior

Printings (paper and PDF) do not contain SVG objects.

### Reproduction Steps

- open a blank document
- insert a SVG
- print to PDF or paper

**Comment by TatianaKirillova** [comment]
Hello @Norbert80! Thank you for your request! 
Unfortunately I can't reproduce the problem. Could you attach a file with the problem image?

**Comment by Norbert80** [comment]
> Hello @Norbert80! Thank you for your request! Unfortunately I can't reproduce the problem. Could you attach a file with the problem image?

Hello @TatianaKirillova,

it looks like some SVG objects are printed and some are not printed. Attached you find a testkit that contains two SVG objects. I have attached the printed PDF too. You can see that the printed PDF contains the ONLYOFFICE logo but not the Reutin logo.

[svg_testkit.docx](https://github.com/user-attachments/files/16108379/svg_testkit.docx)

[svg_testkit.pdf](https://github.com/user-attachments/files/16108381/svg_testkit.pdf)

**Comment by Norbert80** [comment]
Below you find the affected SVG file separately:
![logo_tv-reutin](https://github.com/ONLYOFFICE/DesktopEditors/assets/6921696/2c872cea-5896-4586-bb9a-659a27f12425)

**Comment by TatianaKirillova** [comment]
@Norbert80 oh, thank you very much for such a quick response and information about the problem!

**Comment by TatianaKirillova** [comment]
@Norbert80  Indeed, the problem repeated itself with your image. Issue No. 69024 for your task

The problem is reproduced on Document Server.
